Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: deps
 deps:
-	go install golang.org/x/lint/golint
+	go install golang.org/x/lint/golint@latest
```

Comment:
I knew *golint* is already deprecated.

Though I tried to move golangci-lint instead golint, some example tests were reported naming convention violations for godoc by govet. These violations can be ignored and it is bother to resolve them so I still use golint.